### ISSUE TITLE
sync_unary_ping_pong_day failure

### ISSUE THREAD

**Issue by dgquintas** [grpc/grpc#4024]
```
I1102 12:55:17.169477000 140735209784064 sync_unary_ping_pong_test.cc:51] Running Synchronous Unary Ping Pong
D1102 12:55:17.170353000 140735209784064 test_config.c:155] test slowdown: machine=1.000000 build=1.000000 total=1.000000
E1102 12:55:17.205916000 140735209784064 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
E1102 12:55:17.298843000 140735209784064 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
E1102 12:55:17.369828000 140735209784064 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
E1102 12:55:17.399306000 140735209784064 tcp_client_posix.c:171] failed to connect to 'ipv6:[::1]:32767': socket error: connection refused
D1102 12:55:17.408002000 4448296960 grpc_profiler.c:48] You do not have google-perftools installed, profiling is disabled [for qps_server.prof]
D1102 12:55:17.408029000 4448296960 grpc_profiler.c:52] To install on ubuntu: sudo apt-get install google-perftools libgoogle-perftools-dev
I1102 12:55:17.413520000 140735209784064 driver.cc:206] Warming up
I1102 12:55:22.413665000 140735209784064 driver.cc:212] Starting
E1102 12:55:52.405818000 140735209784064 driver.cc:229] assertion failed: client->stream->Read(&client_status)
```

**Comment by vjpai** [comment]
I believe this flake has been fixed by #4018 . I have run it 100+ times on localhost without repeating the flake. Will watch.

**Comment by vjpai** [comment]
I declare this closed based on a lack of failures in recent master builds since #4018 was merged.
